tests/test_correlation.py

```diff
@@ -200,6 +200,33 @@ def test_synthesize():
                                                      '2017-06-01': 8.525249,
                                                      '2017-08-01': 6.715885,
                                                      '2017-10-01': 9.479016}}
+    result_speed_sort = {'Spd80mN_Synthesized': {'2016-03-01': np.NaN,
+                                                 '2016-04-01': 6.598875,
+                                                 '2016-05-01': np.NaN,
+                                                 '2016-06-01': 5.108156,
+                                                 '2016-07-01': np.NaN,
+                                                 '2016-08-01': 7.093956,
+                                                 '2016-09-01': np.NaN,
+                                                 '2016-10-01': 6.669446,
+                                                 '2016-11-01': np.NaN,
+                                                 '2016-12-01': 8.900778,
+                                                 '2017-01-01': np.NaN,
+                                                 '2017-02-01': 9.134509,
+                                                 '2017-03-01': np.NaN},
+                         'Dir78mS_Synthesized': {'2016-03-01': 257.413204,
+                                                 '2016-04-01': np.NaN,
```

Comment:
The resulting synthesised data has wind speed for 20166-0-01 but there is no wind direction. How can there be wind speed if there is no wind direction?